### DISCUSSION TITLE
Enhance the documentation about command

### DISCUSSION THREAD

**Discussion by Mara-Li** [LostPaul/obsidian-folder-notes#66] [General]
When using the right menu command (3 dots or right click on a file), I fail each time between "Turn into folder note" and "Create folder note". 

My proposition is to improve theses two command to reflect better their purpose :
- `Turn into folder note for {{parentName}}` (not difficult to have with `TFile.parent`
- `Make Folder note` : If the first is edited, i think it will become self explanatory

More over, the documentation needs to be improved for each commands found in the commands palette.

**Comment by berot3** [comment]
Haha same! Im slowly getting used to it. But I agree it's super confusing 🫤

**Comment by LostPaul** [comment]
The documentation has this now since a few weeks, anything else import that is missing?
